lib/transaction.go

```diff
@@ -161,14 +176,19 @@ func (o *Transaction) Sign(kp keypair.KP, networkID []byte) {
 	return
 }
 
-func (o Transaction) NextCheckpoint() string {
-	return string(
-		base58.Encode(
-			sebakcommon.MakeHash(
-				[]byte(fmt.Sprintf("%s%s", o.B.Checkpoint, o.GetHash())),
-			),
-		),
-	)
+// NextSourceCheckpoint generate new checkpoint from current Transaction. It has
+// 2 part, "<head>-<body>".
+//
+// <head>: hash of last paid transaction of source account
```

Comment:
comment should follow https://github.com/bosnet/sebak/issues/60 -- `<subtracted> - <added>`